src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2097,11 +2216,17 @@
           short-summary: Direction of the packet relative to the VM.
         - name: --protocol
           short-summary: Protocol to test.
+    examples:
+        - name: Run test-ip-flow verify to test logical connectivity from a VM to the specified destination IPv4 address and port.
+          text: az network watcher test-ip-flow --resource-group resourceGroupName --direction Outbound  --protocol TCP --local 10.0.0.4:* --remote 10.1.0.4:80 --vm vmName
 """
 
 helps['network watcher test-connectivity'] = """
     type: command
-    short-summary: Test if a direct TCP connection can be established between a Virtual Machine and a given endpoint.
+    short-summary: (Preview) Test if a connection can be established between a Virtual Machine and a given endpoint.
+    long-summary:  |
+                To register for this feature refer to instructions at https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-connectivity-cli
```

Comment:
This uses different language than other `see`-type references, and includes examples, which are not provided anywhere else. Recommend that this is standardized throughout the CLI help. Also, we should have the feature instructions ToCs linking out to the examples anyway, right?

Reply:
Also, check with the service team--this may be GA now and not require feature registration?

Reply:
Appears to still be preview: left as is.